package.json

```diff
@@ -659,7 +659,7 @@
 		"fs-extra": "^8.1.0",
 		"jschardet": "2.1.1",
 		"os-locale": "5.0.0",
-		"vsce": "1.88.0",
+		"vsce": "^2.5.0",
```

Comment:
@Higurashi-kagome 
may I ask what's the reason of this change?

Reply:
Just a change when I try to pack and test it using `vsce`. Ignore it would be good. @Sen-real